scripts/holes.pl

```diff
@@ -0,0 +1,53 @@
+use Irssi;
+use strict;
+use warnings;
+
+our $VERSION = "1.0.0";
+our %IRSSI = (
+    authors     => 'terminaldweller',
+    contact     => 'https://terminaldweller.com',
+    name        => 'holes',
+    description => 'gives a list of of the open sockets as an expando(this makes sense only if irssi is in a container)',
+    license     => 'GPL3 or newer',
+    url         => 'https://github.com/irssi/scripts.irssi.org',
+);
+
+Irssi::settings_add_int('misc', 'holes_frequency', 30000);
+Irssi::settings_add_str('misc', 'holes_separator', '');
+my $holes = "";
+my $timeout;
+my $holes_cmd = << 'HOLES_CMD';
+lsof | grep socket | awk '{print $4}' | awk 'BEGIN{FS=":"}{print $2}' | tr -d [] | uniq
+HOLES_CMD
+
+sub uniq {
+    my %seen;
+    grep !$seen{$_}++, @_;
+}
+
+sub holes_sub {
+    my $result;
+    Irssi::timeout_remove($timeout);
+    my $output = `$holes_cmd`;
```

Comment:
I dont expect this to take long since running this script only makes sense if you are running it inside an applicatino container. The comments explicitly mention that.
But generally, you're right.